cmd/better-retester/retester.go

```diff
@@ -90,6 +82,31 @@ func findCandidates(config config.Getter, gc githubClient, usesGitHubAppsAuth bo
 	return prs, nil
 }
 
+func (c *retestController) atLeastOneRequiredJob(candidates map[string]tide.PullRequest) (map[string]tide.PullRequest, error) {
```

Comment:
This is the function that I failed to extract from tide.
I really tried, but failed.